src/test/regress/expected/multi_partitioning_1.out

```diff
@@ -541,6 +541,18 @@ SELECT tablename, indexname FROM pg_indexes WHERE tablename LIKE 'partitioning_t
 -----------+-----------
 (0 rows)
 
+-- test drop
+-- indexes created on parent table can only be dropped on parent table
+-- ie using the same index name
+-- following will fail
+DROP INDEX partitioning_test_2009_id_idx;
+ERROR:  index "partitioning_test_2009_id_idx" does not exist
+-- but dropping index on parent table will succeed
+DROP INDEX partitioning_index;
+ERROR:  index "partitioning_index" does not exist
+-- this index was already created on partition table
+DROP INDEX partitioning_2009_index;
+ERROR:  index "partitioning_2009_index" does not exist
```

Comment:
fix looks fine, but we haven't covered the original problem in the tests: dropping an index on a non-distributed partitioned table throws an error before this fix.